app/controllers/application_controller.rb

```diff
@@ -15,11 +15,12 @@ def configure_permitted_parameters
   private
 
   def setup_offcanvas_menu
+    @menu_current_battles   = Battle.unwon.order(:created_at)
     #TODO: Refactor @menu_scores to be more performant. Calling User.all.sort_by is not scalable.
     @menu_scores            = User.all
                                   .sort_by{|a,b| a.battles_won.count <=> b.battles_won.count if [a,b].all?}
                                   .first(5)
-    @menu_battles           = Battle.order(:created_at)
+    @menu_recent_battles    = Battle.won.order(:created_at)
```

Comment:
Something that I was playing with was
```ruby
Battle.won.order(created_at: :desc).first(5)
```

On the surface, this is a trivial difference, but it's differentiating factor is that while the same five battles are selected, my method presents them in newest-to-oldest order. A matter of preference, really.